Adjust documentation for event system after recent API changes (#5224)
microsoft/MixedRealityToolkit-Unity: Documentation/Input/Pointers.md

Pointer Down / Dragged / Up / Clicked | Raised to report pointer press, drag and release. | [`IMixedRealityPointerHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler)
Touch Started / Updated / Completed | Raised by touch-aware pointers like [**PokePointer**](xref:Microsoft.MixedReality.Toolkit.Input.PokePointer) to report touch activity. | [`IMixedRealityTouchHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityTouchHandler)

> To handle them you must implement the corresponding interface in your script. Aditionally, if you want your game object to receive the events raised on all game objects instead of just the ones raised on itself and its children, you must register as a global event listener calling [`RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem.RegisterHandler*) on the input system with the interface(s) you want to receive events for.

Reviewer: ```suggestion
To handle them the corresponding interface must be implemented. Additionally there's the option to receive the events raised on all game objects instead of just the ones on itself and its children. This requires to register as a global event listener calling [`RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem.RegisterHandler*) on the input system with the corresponding interface(s).
```

avoid you

> > [!WARNING]

Reviewer 2: We could downgrade this to just a note saying that those types of events should not be marked as used by global handlers.

> > [`IMixedRealityFocusChangedHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusChangedHandler) and [`IMixedRealityFocusHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusHandler) should be handled in the objects they are raised on. It is possible to receive focus events globally but, unlike other input events, global event handler won't block receiving event based on focus (the event will be received by both global handler and a corresponding object in focus).

Reviewer: ```suggestion
> [`IMixedRealityFocusChangedHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusChangedHandler) and [`IMixedRealityFocusHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusHandler) should be handled in the objects they are raised on. It is possible to receive focus events globally but, unlike other input events, global event handler won't block receiving events based on focus (the event will be received by both global handler and a corresponding object in focus).
```

> Alternatively, you can derive your handler script from [`InputSystemGlobalHandlerListener`](xref:Microsoft.MixedReality.Toolkit.Input.InputSystemGlobalHandlerListener) if you require global registration or from [`BaseInputHandler`](xref:Microsoft.MixedReality.Toolkit.Input.BaseInputHandler) if you want to choose in the inspector whether to register as global listener or not. In both cases you need to implement abstract methods `RegisterHandlers` and `UnregisterHandlers`, in which specify what interfaces you want to listen global events for.

Reviewer: ```suggestion
Alternatively, there's the possibility to derive the handler script from [`InputSystemGlobalHandlerListener`](xref:Microsoft.MixedReality.Toolkit.Input.InputSystemGlobalHandlerListener) for global registration or from [`BaseInputHandler`](xref:Microsoft.MixedReality.Toolkit.Input.BaseInputHandler) for choosing in the inspector whether to register as global listener or not. In both cases the abstract methods `RegisterHandlers` and `UnregisterHandlers` which specify the interfaces to listen to, need to be implemented.
```
